cliStocksTracker.py

```diff
@@ -411,6 +343,51 @@ def print_graphs(self):
             graph.draw()
         return
 
+    def print_gains(self, format_str, gain, timespan, mode):
+        positive_gain = gain >= 0
+        gain_symbol = "+" if positive_gain else "-"
+        gain_verboge = "Gained" if positive_gain else "Lost"
+
+        print("{:25}".format("Value " + gain_verboge + " " + timespan + ": "), end="")
+        print(Fore.GREEN if positive_gain else Fore.RED, end="")
+        print(
+            format_str.format(
+                gain_symbol + "$" + str(abs(utils.round_value(gain, mode, 2)))
+            )
+            + format_str.format(
+                gain_symbol
+                + str(
+                    abs(utils.round_value(
+                        gain / self.current_value * 100, mode, 2
+                    ))
+                )
+                + "%"
+            )
+        )
+        print(Style.RESET_ALL, end="")
+        return
+
+    def print_portfolio_summary(self, format_str, table):
+        for line in table:
+            if line[-1] is None:
+                pass
+            else:
+                print(Fore.GREEN if line[-1] else Fore.RED, end="")
+
+            print("\t" + "".join([format_str.format(item) for item in line[:-1]]))
+            print(Style.RESET_ALL, end="")
+
+        # print the totals line market value then cost
+        print("{:112}".format("\nTotals: "), end="")
+        print(Fore.GREEN if self.current_value >= self.initial_value else Fore.RED, end="")
+        print(format_str.format("$" + str(round(self.current_value, 2))), end = "")
+        print(Fore.RESET, end="")
+        print(
+            "{:13}".format("")
+            + format_str.format("$" + str(round(self.initial_value, 2)))
+        )
+        return
+
```

Comment:
same as previous PR

Reply:
Still seeing those native round() functions! Looking for utils.round() so the user can choose if they are using the expected round() action or if it should always round down.